h2o-py/tests/testdir_algos/gbm/pyunit_gbm_PUBDEV_8276.py

```diff
@@ -15,7 +15,7 @@ def test_weights_column_not_in_train():
         gbm.train(y=-1, training_frame=df)
         assert False, "Model building should fail."
     except H2OResponseError as e:
-        assert "ERRR on field: _weights_column" in e, "Model building should fail with this in message."
+        assert "ERRR on field: _weights_column" in e.__str__(), "Model building should fail with this in message."
```

Comment:
```suggestion
        assert "ERRR on field: _weights_column" in str(e), "Model building should fail with this in message."
```

Reply:
Thanks, @sebhrusen! I had an eclipse of the brain. :D